quadratic/quadratic_problem.py

```diff
@@ -65,6 +67,87 @@ def from_params(cls, G: np.ndarray, c: np.ndarray, A: np.ndarray,
                    solution=solution, x0=x0)
 
     @property
-    def is_inequality_constrained(self):
+    def is_inequality_constrained(self) -> bool:
         """Return whether or not any of the imposed constraints are inequalities."""
-        return any(constraint.equation_type is not EquationType.EQ for constraint in self.constraints)
+        return any(constraint.equation_type is not EquationType.EQ
+                   for constraint in self.constraints)
+
+    def find_x0(self, initial_guess: np.ndarray = None) -> np.ndarray:
+        """Find initial solution
+
+        Args:
+            initial_guess: Point to start from.
+
+        Returns:
+            np.ndarray - Feasible point.
+        """
+        if self.x0 is not None:
+            return self.x0
+        if initial_guess is None:
+            return find_x0(self, standardized=False)
```

Comment:
i glaub, für initial_guess=0, wäre des desselbe wie find_x0
i finds sowieso schiach, dass ma des standardized bei find_x0 so übergeben miasn, evtl könnt ma den parameter und den dazugehörigen code ganz loswerdn wenn wir hier initial_guess einfach auf 0 setzn.

Reply:
Fair, ma könnt des ganze in 1 fassen